aws/PAS-network-environment-NAT.json

```diff
@@ -742,8 +742,8 @@
                     "Ref": "VaultSG"
                 },
                 "IpProtocol": "icmp",
-                "FromPort": "-1",
-                "ToPort": "-1",
+                "FromPort": -1,
```

Comment:
I assume that all that "" that were removed were as a result of the cfn-lint?

Reply:
True. Currently ran it only in 1 template. We should fix it gradually